pyproject.toml

```diff
@@ -32,7 +32,7 @@ rasterio = { version = "^1.3.4", optional = true }
 dask-geopandas = { version = ">=0.2.0,<1", optional = true }
 xgboost = { version = ">=1.5.1", optional = true }
 rioxarray = { version = ">=0.12.0,<1", optional = true }
-openeo-pg-parser-networkx = { version = ">=2023.5.1", optional = true }
+openeo-pg-parser-networkx = "2024.7"
```

Comment:
The current version of openeo-processes-dask needs the openeo-pg-parser-networkx to be 2024.7, but versions above that one should be allowed, too. 
You could update the file with 
```
openeo-pg-parser-networkx = { version = ">=2024.7", optional = true }
```

Reply:
I don't think the example%20file.geoparquet is required.

Reply:
okay - then please delete the file within your nextcommit :)